src/components/Table/Table.tsx

```diff
@@ -656,7 +664,7 @@ function Table<Values extends Record<string, unknown>>({
                 const classes = classNames({
                   'table-line': true,
                   clickable: onRowClick !== NOP || isExpandable,
-                  'is-expand': extendedRow.isExpanded,
+                  'is-expand': !extendedRow.isGrouped && extendedRow.isExpanded,
```

Comment:
I'm not quite sure - if the row is completely expanded and we're coming to the JSON part (like in Servers page) - don't we expect the whole group to have the border?
![Screen Shot 2023-06-19 at 10 37 06](https://github.com/weka/weka-ui-components/assets/63452816/01e341b8-4c06-45a5-9fb5-caf913847966)

Reply:
Jacky said leave it for now